.github/workflows/long-running-azure.yaml

```diff
@@ -45,40 +45,55 @@ on:
     branches:
       - main
     paths:
-      - '.github/workflows/long-running-azure.yaml'
+      - ".github/workflows/long-running-azure.yaml"
 
 env:
   # Go version
-  GOVER: '^1.21'
+  GOVER: "^1.21"
   GOPROXY: https://proxy.golang.org
 
   # gotestsum version - see: https://github.com/gotestyourself/gotestsum
-  GOTESTSUM_VER: 1.10.0
+  GOTESTSUM_VER: 1.11.0
+
+  # Helm version
+  HELM_VER: "v3.13.0"
+  # KinD cluster version
+  KIND_VER: "v0.20.0"
+  # Dapr version
+  DAPR_VER: "1.12.0"
+  DAPR_DASHBOARD_VER: "0.14.0"
+  # Kubectl version
+  KUBECTL_VER: "v1.29.0"
+  # Azure Keyvault CSI driver chart version
+  AZURE_KEYVAULT_CSI_DRIVER_VER: "1.5.0"
+  # Azure workload identity webhook chart version
+  AZURE_WORKLOAD_IDENTITY_WEBHOOK_VER: "1.2.0"
```

Comment:
Is this temporary change? We do not need all of them.

Reply:
Removed all.